stdlib/stdlib-external/ascutils/dataStructures.ts

```diff
@@ -68,6 +68,26 @@ export function allocateString(size: u32): u32 {
   return str
 }
 
+export function singleCharacterString(char: u8): u32 {
+  let s = allocateString(1)
+  store<u8>(s + 8, char)
+  return s | GRAIN_GENERIC_HEAP_TAG_TYPE
+}
+
+export function twoCharacterString(char1: u8, char2: u8): u32 {
+  let s = allocateString(2)
+  store<u8>(s + 8, char1)
+  store<u8>(s + 8, char2, 1)
+  return s | GRAIN_GENERIC_HEAP_TAG_TYPE
+}
```

Comment:
Could you call these `singleByteString` and `twoByteString`? I'm currently adding a `Char` type to Grain (which can be up to four bytes) and calling this character could get confusing (especially since this is in the `dataStructures` module).